src/test/kotlin/no/nav/syfo/aktivitetskrav/api/AktivitetskravApiSpek.kt

```diff
@@ -384,6 +401,42 @@ class AktivitetskravApiSpek : Spek({
                             response.status() shouldBeEqualTo HttpStatusCode.BadRequest
                         }
                     }
+                    it("should return status BadRequest if vurdering missing arsak") {
```

Comment:
Fine testnavn! 🥇

Reply:
+1

Reply:
Takk, men ser nå at jeg kan droppe "should" foran 😄